.codecov.yaml

```diff
@@ -0,0 +1,25 @@
+# https://docs.codecov.com/docs/codecovyml-reference
+coverage:
+  precision: 2
+  round: down
+  range: 39..75
+
+  status:
+    project:
+      default:
+        target: auto # automatically calculate coverage target - should increase
+        threshold: 2% # allow for 2% reduction without failing
```

Comment:
Is there an example run of the gating taking effect? It would be useful to learn what to expect.
If not, we should create a dummy PR with a bunch of uncovered code to see after this change goes in.
Is 2% global or per file? If global my inkling is that it is pretty high, how about 1% if we have to stick with integers?
It's a difference between thousands of lines of code.

Reply:
Updating to 1%